Eye Changes for Ethnicities

ercarp opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Gave **Draenei** `magic_no_pupils_eyes` and changed the brightness to 0.5.
- ~~Gave **Frostborn Dwarves** `magic_no_pupils_eyes`~~ and changed the brightness to 0.5.
  - Reverted back to `magic_eyes` in 997e635ab0489d202845889b9a3f9c706de1039a.
- Gave **Gilblin** and **Ice Trolls** `glowing_iris` and changed brightness to 1.
- Gave **Frost Vrykul**, **Kvaldir** and **Drust** `magic_no_pupils_eyes` and changed their brightness to 0.5.
- Readjusted elf eye brightness to 0.5, down from 1.
- Readjusted troll eye brightness to 0.5, up from 0.35.
- Readjusted **Frost Vrykul** and **Drust** eye color ranges so they include slightly lighter variations.

<details><summary>Images</summary>

![image](https://user-images.githubusercontent.com/78575425/129946859-f19c4b3b-4d0a-4abd-8daf-0ffb3c89614a.png)
![image](https://user-images.githubusercontent.com/78575425/129946905-4b85af6a-5984-4f11-a1c5-c8d52ad8969b.png)
![image](https://user-images.githubusercontent.com/78575425/129947071-65bbf8e2-0f08-4127-9a84-41fed64a5dca.png)
![image](https://user-images.githubusercontent.com/78575425/129947023-549f181f-2d37-4552-be10-799c9324c48c.png)
![image](https://user-images.githubusercontent.com/78575425/129947263-1ba4e7bc-2fb6-4c39-a523-2fdb6c01c302.png)
![image](https://user-images.githubusercontent.com/78575425/129947298-6deec33f-4569-4eda-b49b-e2564ea315cd.png)
![image](https://user-images.githubusercontent.com/78575425/129947364-8389b451-01e3-4ef9-8924-268d8421ebde.png)

</details>

### Implemented Suggestions
- Gave **Ice Trolls** more eye color variation in e96b155526bca4f9366f9a99cb0285b3fceb08e2 (suggested by @Remerod).
  - They may now generate with blue, cyan, purple, and pink eyes in addition to red and orange eyes. Red and orange eyes are, by far, the most common variation.
- Changed **Ice Troll** eyes to glowing_iris in 04f0e888262418b9761b83b56f0147bd712bf86f (suggested by @Remerod).
- Changed **Gilblin** eyes to glowing_iris 04f0e888262418b9761b83b56f0147bd712bf86f (suggested by @Remerod).

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
- View the changed ethnicities either using the Ruler Designer, [Debug Portrait Editor](https://ck3.paradoxwikis.com/Console_commands#Enabling_debug_mode), or by looking for their cultures in-game with the Character Finder (Draenei and Gilblin cannot be found using the last method).
- Provide feedback on the new eyes and if you think any other ethnicities should have their eyes changed or altered in some way (colors, brightness, eye type, etc.). You may use [this album](https://imgur.com/a/pQKUdkd) to make comparisons.

ValianBlue reviewed (approved):
_Overall the changes look great. The brightness adjustment to the elven eyes are my favorite in this change. They look more natural and fitting for their type._

_I'm curious about how the `magic_no_pupils_eyes` type would look with smaller pupils. This is not an issue, however._

zumbak04 commented on common/ethnicities/wc_ethnicities_elven.txt:
```diff
@@ -370,7 +370,7 @@ high_elven = {
 	#Eyes
 	special_eyes = {
-        1 = { name = magic_eyes      range = { 1 1 } }
+        1 = { name = magic_eyes      range = { 0.5 0.5 } }
```
The glow of the DNA elves Kael'thas, Anasterian has not changed. Is it inteted?

ercarp replied:
Wasn't intended. Fixed in 617d9e53e1df8cb0acac34d0f0790dbcf08923f1.

Remerod reviewed (changes requested):
So Overall i really like this change. but i do Have some comment's on the eye's

**Frostborn**
<details>
<summary>Click to expand</summary>

I think the Frostborn dwarf should have glowy eyes, but less so or maybe a change to their eye's, becouse currently they look a little souless to me.
![image](https://user-images.githubusercontent.com/79544635/130298685-d29d43fd-bee3-4d29-9b5c-f769d61554ef.png)
![image](https://user-images.githubusercontent.com/79544635/130298702-a3e3acb9-7806-4316-be96-c5eea839911d.png)
Here is one on Falstad from HoTS with frostbornskin, i think his eye's look more alive
![image](https://user-images.githubusercontent.com/79544635/130298899-e53262a0-b9d1-4394-80b3-25aa6c6abe2c.png)
Generally Frostborn in WoW looks like they have a glow, but really a small one
![image](https://user-images.githubusercontent.com/79544635/130299035-6faed2b7-1866-4642-bb0e-b5753e987efd.png)
I am honestly starting to think that the old eyes looked great
![image](https://user-images.githubusercontent.com/79544635/130299258-cfb3b798-be0d-42cb-a635-d5bd36c48ac7.png)

It is really hard to estimate what is a good eye for frostborn, since their is almost zero official art on them and the models are oldish. I do feel like their eye (as in eye white) is darker (blue) then their iris, who seem to be brighter and white-blueish. So perhaps something in between of the old and new eyes for them is better with darker eye and a lighter iris/pupil? 

</details>

On the topic of  **Frost Vrykul, Kvaldi**r and **Drust** , I think they look great! they look very eerie/spooky and it fit their theme very well! 

**Zandalari troll's** look great!

**Ice Troll**
<details>
<summary>Click to expand</summary>

I still is on the opinion that Ice Troll need's an pupil, not just an iris, as they currently have. Our version fit's the old wow models for Ice trolls, but the most modern Ice troll, King Malakk and newer artwork have them with not with just irises but with actual pupil's as well.
![image](https://user-images.githubusercontent.com/79544635/130300160-e55c9b9f-54e3-4b20-8e88-499912e2d58c.png)
![image](https://user-images.githubusercontent.com/79544635/130300217-17b8b7dc-1f72-4cee-86a1-916a4568dc49.png)
![image](https://user-images.githubusercontent.com/79544635/130300427-d1fe540a-7272-482f-87d3-5d42224ebe42.png)
Put then again, there are some artwork with them having just irises and no pupil's
![image](https://user-images.githubusercontent.com/79544635/130300565-05c0aec5-8f9e-4511-90ad-96e542b92575.png)
And then we have Warcraft 3 reforged stuff, where they seem to have only blue eye's. But still with both Irises and Pupils together. 
![image](https://user-images.githubusercontent.com/79544635/130300694-940cc947-860d-42d1-93dd-ba0b63923633.png)
![image](https://user-images.githubusercontent.com/79544635/130301069-b70d2b04-0e3b-44db-bb38-142151c1adb8.png)
![image](https://user-images.githubusercontent.com/79544635/130301171-9e97d85f-2b75-4fd0-8154-156d828c8333.png)
So it could be nice to add some eye colours perhaps? but mostly I'd like to see how they look with both pupil's and irises together, because they look nice as they are now but they stick out from the other non-zandalari trolls when they shouldn't, cause they are not more magical then any other troll kind.

</details>

**Gilblin**
<details>
<summary>Click to expand</summary>

Almost every Gilblin I have seen have both Irises and blackish pupil's in them, So i am unsure why ours don't.
![image](https://user-images.githubusercontent.com/79544635/130301699-f817b7b9-e3a0-4ff2-83dc-14f7ca9d6b27.png)
![image](https://user-images.githubusercontent.com/79544635/130301707-4680e435-13f9-49cb-a103-91b5861c1944.png)
![image](https://user-images.githubusercontent.com/79544635/130301721-6448dc97-474b-487e-8879-9dd45693722d.png)
![image](https://user-images.githubusercontent.com/79544635/130301741-585abdf3-aa67-4a50-8c67-a0528f80eafb.png)
![image](https://user-images.githubusercontent.com/79544635/130301748-4e286829-dbaa-47bd-b1e5-4bba1cc42aec.png)
![image](https://user-images.githubusercontent.com/79544635/130301758-48c32e82-f67a-436e-87ce-d4e0b6e0d680.png)
![image](https://user-images.githubusercontent.com/79544635/130301766-cfa9b7fc-5c21-43e3-9408-2ce4eac5087c.png)
Some of them do on the other hand have coloured eye-white. So i feel like they could use a change.

</details>

**Daenei Eredar** looks great and so does **Man'ari Eredar**

Provide feedback on the new eyes **and if you think any other ethnicities should have their eyes changed or altered in some way**

**Night Elf/Kaldorei**
<details>
<summary>Click to expand</summary>

So the Night Elf in game looks great at the moment! And i don't think they need to change... **But** in my opinion they **can** become better. Currently then look really close how they do in WoW. But we do know that how they look in WoW is not how Blizzard officialy depict them in their more detailed work. 
![image](https://user-images.githubusercontent.com/79544635/130302178-f356685d-7a14-4aec-a9c2-a3e1cf56ec6a.png)
![image](https://user-images.githubusercontent.com/79544635/130302238-e846ddba-f9b8-47fe-a736-15f7c563b29e.png)
As stated, they look good, and close to the game version.
But i think most "great" art conception of Night Elf have more variation and nuance in their eyes.
![image](https://user-images.githubusercontent.com/79544635/130302398-3df67925-24eb-4711-a3b5-5beb94d687a7.png)
It is hard to see, but there is an eye-white, but not shown in this picture but the eye-white do also often have a slight glow, there is the iris that is very bright, and this is the hard to see part, the pupil is outlined here, it is of course not black but kind of the same colour of the iris, other picture have the pupils more brightly glowing then the iris.
![image](https://user-images.githubusercontent.com/79544635/130302528-e0adf0dc-a6ca-4482-b04c-7f439e348950.png)
On this wc3 concept art you can clearly see a beautiful moonlight glow from the pupil, with a less but stil shining silvery moonish Iris and with a less but faintly glowing eye-white.
![image](https://user-images.githubusercontent.com/79544635/130303287-28fa08ed-2060-4e96-9475-00a6af3b0b03.png)
![image](https://user-images.githubusercontent.com/79544635/130302692-965d12e1-bb42-4452-94eb-1d910ebd6d9e.png)
![image](https://user-images.githubusercontent.com/79544635/130302848-b75a151c-7953-48f1-b2a8-73f5441f6090.png)
![image](https://user-images.githubusercontent.com/79544635/130302859-132b64b3-8eb0-4ad5-b89d-fe7affb07c18.png)
![image](https://user-images.githubusercontent.com/79544635/130302876-b45a09ce-a5c7-4b23-9b3c-fb227fcf26e0.png)
some female nelfs
![image](https://user-images.githubusercontent.com/79544635/130302755-8f71cbcb-3063-453d-aa6a-018cdf69f3c3.png)
![image](https://user-images.githubusercontent.com/79544635/130302772-b5ff6554-3f63-4de0-93af-ed0764ad575b.png)
While druid's clearly have different eyes, since they are supposed to be affected with druid magic. 
![image](https://user-images.githubusercontent.com/79544635/130302956-118f6b1c-7ae9-4c8f-8b8f-e273d9bb0c05.png)
![image](https://user-images.githubusercontent.com/79544635/130303051-40ea7de0-7c41-44a1-8e68-3d6fdc707174.png)
Malfurion's eyes
![image](https://user-images.githubusercontent.com/79544635/130303151-2b8a9773-89b2-4ab4-b043-6650fcb36fe4.png)
![image](https://user-images.githubusercontent.com/79544635/130303164-8403b15c-9675-43dd-bf8b-b26ef59e996f.png)
![image](https://user-images.githubusercontent.com/79544635/130303181-4c4707bd-a397-4b2b-91b6-4499ad538d1f.png)
Tyrande's eyes.

And I don't think the NIght Elf's are the only only example elf's you could see a shiny pupil in their iris. Some High/Blood elf's actually is seen with dark pupil's in their art and wc3 stuff.
![image](https://user-images.githubusercontent.com/79544635/130303512-b13e9726-d51d-4198-8d8e-19a34eb60a24.png)
![image](https://user-images.githubusercontent.com/79544635/130303523-64af9908-17b5-4dfe-99c9-dea20e67d839.png)
![image](https://user-images.githubusercontent.com/79544635/130303557-7c5e62f2-14be-4c02-8011-f4419c204d0a.png)
![image](https://user-images.githubusercontent.com/79544635/130303580-cf483066-3b4d-41eb-a37b-4899b859c536.png)
![image](https://user-images.githubusercontent.com/79544635/130303679-c4c079c9-e53e-47e0-94ed-d10a642a69b1.png)
![image](https://user-images.githubusercontent.com/79544635/130303766-1f7020b3-dfcf-44e6-b946-9ad19cdbb5a1.png)
![image](https://user-images.githubusercontent.com/79544635/130303798-03ed389d-bd25-462f-b2a5-3c69958b9f27.png)
![image](https://user-images.githubusercontent.com/79544635/130303808-0f359cb4-ca76-4327-911b-b1146cdc9914.png)
![image](https://user-images.githubusercontent.com/79544635/130303821-7453ea6b-4363-4fb6-a2bb-7251b5de89c6.png)
![image](https://user-images.githubusercontent.com/79544635/130303929-80515e76-2377-45ae-902d-7e2351c08970.png)
![image](https://user-images.githubusercontent.com/79544635/130303844-aabc3377-bdcd-4a60-a7af-1a653a828e67.png)
![image](https://user-images.githubusercontent.com/79544635/130303848-772c1006-f5eb-458a-a18b-8bae050406ec.png)
So there are plenty examples.

![image](https://user-images.githubusercontent.com/79544635/130304277-9b353b4d-6706-4379-ac1c-ce6e78f0be0a.png)
So I would like to suggest for a edited/possible new eye type for normal silver nelf eye's to be like in this picture that, 1. have a brighter light source as/(or over) a pupil, 2. have a less but still shining iris, that is not the same light as the pupil, but more silverish. 3 have the eye-white also have a slight glow but very feint. So all 3 should look from a distance like they meld together but if you look closely it all is different.

As i pointed out earlier, what we have now is good, it fit well with the wow/game look. And it is really close to what i am suggesting, but i feel if we could differentiate the different eye part's more for Night elves or elves in general it would look a little better in my opinion if we want them to look more realistic, or more in line of what Blizzard envisioned their eye's to be like.

But i stress out again, that this is just a extra suggestion.

</details>

ercarp commented:
> I think the Frostborn dwarf should have glowy eyes, but less so or maybe a change to their eye's, becouse currently they look a little souless to me.

Well, you've clearly done your homework on this! I'll change them back to what they had before, you make some good points. The current ones may be better reserved for the **_Earthen_**:

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/130306838-4a8385e8-dfc6-4974-8971-2f95ae91962b.png)

</details>

> So it could be nice to add some eye colours perhaps? but mostly I'd like to see how they look with both pupil's and irises together, because they look nice as they are now but they stick out from the other non-zandalari trolls when they shouldn't, cause they are not more magical then any other troll kind.

I will add more eye color options for Ice Trolls, but I will make them rarer than the red eyes (which seem to be the most common, I'd say).

> I still is on the opinion that Ice Troll need's an pupil, not just an iris, as they currently have. Our version fit's the old wow models for Ice trolls, but the most modern Ice troll, King Malakk and newer artwork have them with not with just irises but with actual pupil's as well.

> Almost every Gilblin I have seen have both Irises and blackish pupil's in them, So i am unsure why ours don't.

I will ask @zumbak04 about adding this new type of eye. I think it would be good to have this as an option.

> So the Night Elf in game looks great at the moment! And i don't think they need to change... **But** in my opinion they **can** become better. Currently then look really close how they do in WoW. But we do know that how they look in WoW is not how Blizzard officialy depict them in their more detailed work.

What you're suggesting is nice, but it is very similar to magic_eyes. Therefore it may not be resource- or time-efficient to add a detail that would be barely visible in the character panel. You would only notice something like this when zooming in on the character's face.

ercarp commented:
@Remerod **Gilblin** and **Ice Trolls** now use the new `glowing_iris` eyes.

<details><summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/78575425/130587711-fb06fcb5-cfe2-4c91-85fe-66d771cbd175.png)
![image](https://user-images.githubusercontent.com/78575425/130587732-2ed9872a-082d-4572-8a22-2746083f90d7.png)
![image](https://user-images.githubusercontent.com/78575425/130587753-8a4053ea-7988-4c1b-8d12-f7be28f126a7.png)
![image](https://user-images.githubusercontent.com/78575425/130587792-1d0539f7-4584-46a0-a52c-1f54d0d3c33d.png)

</details>

Remerod reviewed (commented):
I really like the new eye changes!!! They fit so well.

But i do have some remarks.
<details>
<summary>Click to expand</summary>

I think it looks really well with the new eye colours to!
![image](https://user-images.githubusercontent.com/79544635/130684769-aaaf9902-93ce-44b0-a447-8e833b3f9749.png)

But i dunno if the red eye's disappeared? or if they just look more orange now, some are clear orange but other's are more orange red.
![image](https://user-images.githubusercontent.com/79544635/130684930-a066b4af-85a6-491c-bf65-0b6bccc51195.png)
![image](https://user-images.githubusercontent.com/79544635/130686151-c35f1576-9da1-42bc-8845-a25f668ecee6.png)

But perhaps that fit's their looks?
![image](https://user-images.githubusercontent.com/79544635/130685569-12ff7331-698c-4ef4-a04d-b3a89ccd1caf.png)
So is it intended to be less red? because it was really red before, i don't really dislike the new lessened red.

Also King Malak have not gotten is new eye's, "sadge"
![image](https://user-images.githubusercontent.com/79544635/130685895-95c264b5-9fc5-4720-9e3c-6d6fb8599039.png)

The Gilblin look's good with their new eye's
![image](https://user-images.githubusercontent.com/79544635/130687226-b012cf27-400e-4931-b5b6-b68d28b4fbc5.png)
But dare i say where they are rich in quality they lack in quantity . I can only see the yellow-orange-ish & the blue eye colours in their randomnize appearance on the character design. and while it is not canon,  I would suggest to add this green-yellowish eye colour as a third normal appearance for Gilblin, found on google image search of Gilgoblin.
![image](https://user-images.githubusercontent.com/79544635/130687867-7f491f55-c912-4449-b2e9-861f0459be8f.png)

Overall i really like this change, good work!
</details>

zumbak04 commented on common/dna_data/wc_elf_dna.txt:
```diff
@@ -106,7 +106,7 @@ lorthemar_theron_dna = { # by ercarp & TheDrPibb
 			eyelashes_accessory={ "normal_eyelashes" 181 "normal_eyelashes" 181 }
```
You haven't changed the fel orc and undead eyes. Is it intended?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/130708132-4cec1fad-f45b-4867-aa8d-acadaf1afc3f.png)
![image](https://user-images.githubusercontent.com/46384992/130708167-0514ac18-e5a2-4d7b-b907-c586ee3d628e.png)

</details>

ercarp replied:
Wasn't intended. Fixed in aec3c90b77bb3a5731ae05ba442101b2fd103790.

<details><summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/78575425/130722659-e6b84f9e-98b7-4b63-9b5c-93ed8c9b268e.png)
![image](https://user-images.githubusercontent.com/78575425/130722667-3b9a67a8-0948-43f9-a3f9-0b68ad615e1a.png)

</details>

ercarp commented:
> But i dunno if the red eye's disappeared? or if they just look more orange now, some are clear orange but other's are more orange red.
> So is it intended to be less red? because it was really red before, i don't really dislike the new lessened red.

It wasn't intended, it's just that colors interact slightly differently with the new eyes so they look a bit more toned down in terms of colors. But as you said, I don't think it's a bad thing necessarily.


> Also King Malak have not gotten is new eye's, "sadge"

Thanks for pointing that out, added in aec3c90b77bb3a5731ae05ba442101b2fd103790.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/78575425/130721565-de877490-bea9-43fe-b31e-fddf96ea427a.png)

</details>

> But dare i say where they are rich in quality they lack in quantity . I can only see the yellow-orange-ish & the blue eye colours in their randomnize appearance on the character design. and while it is not canon, I would suggest to add this green-yellowish eye colour as a third normal appearance for Gilblin, found on google image search of Gilgoblin.

Added in aec3c90b77bb3a5731ae05ba442101b2fd103790. Gilblin may now generate with green/yellow eyes as well. I also bumped up the chance for them to generate with eyes other than red/orange.

<details><summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/78575425/130721231-8f8c2348-1eec-4fd2-8213-1fcc633da28e.png)
![image](https://user-images.githubusercontent.com/78575425/130721244-a4236e43-94fb-4cf3-8d00-fd6081ceac9c.png)

</details>

zumbak04 reviewed (approved):
LGTM! Good job!
